tests/xfstests/run.pm

```diff
@@ -451,6 +458,7 @@ sub run {
                 collect_fs_status($category, $num);
                 if (get_var('BTRFS_DUMP', 0) && (check_var 'XFSTESTS', 'btrfs')) { dump_btrfs_img($category, $num); }
             }
+            if (get_var('RAW_DUMP', 0)) { raw_dump($category, $num); }
```

Comment:
I recommend this line move to the if structure when`$status =~ /FAILED/`. Only rawdump when test fail, I see all pass tests has this rawdump.

Reply:
Considering this has parameter to control it's OK to leave this out of if structure.